keps/sig-windows/20190103-windows-node-support.md

```diff
@@ -199,6 +201,14 @@ tolerations:
       effect: "NoSchedule"
 ```
 
+#### Memory Overprovisioning
+
+Windows always treats all user-mode memory allocations as virtual, and pagefiles are mandatory. The net effect is that Windows won't reach out of memory conditions the same way Linux does, and processes will page to disk instead of being subject to out of memory (OOM) termination. There is no way to guarantee a physical memory allocation or reserve for a process. See [#73417](https://github.com/kubernetes/kubernetes/issues/73417) for more details on the investigation for 1.14.
```

Comment:
> There is no way to guarantee a physical memory allocation or reserve for a process.

When limits are set I did see the a hard limit.  I used [sysinternals testlimit.exe](https://live.sysinternals.com/WindowsInternals/) to slowly leak memory and found it hit the limit set on the container.  For example I was able to set 100mb limit not go beyond it.

Reply:
clarified that only limits are enforceable